### ISSUE TITLE
Add to existing subscription

### ISSUE THREAD

**Issue by mazgor** [Prospress/woocommerce-subscribe-all-the-things#334]
(1) Placement of tick box:
Storefront theme. If there is already a subscription and the user is logged in and wants to buy a new product and therefore hits sign up, than the product is added to the cart where the customer can choose interval, amount etc., but can't add it anymore to an existing subscription. Only if the customer wouldn't have hit the sign up button but clicked on the product to get to the product view, than there would have appeared the tick box where he can add it to an existing subscription.
(2) If the customer is in the product view an sees the tick box 'add to an existing subscription', than he might tick the box, but maybe hasn't chosen the desired interval yet, but if he keeps the box ticked and changes the interval, no matching subscription will appear, it seems, the system only runs the check once (the first time) if interval is changed, no matches are displayed (because no search is initiated) only by refreshing the site and having the right interval already chosen, than the matching subscription appear. Even 'worse' if customer knows he has a matching subscription but the subscription does not appear although the box is ticked, he maybe thinks, he has to proceed, klicks the sign up and the product goes to the cart, but a new subscription is created than...

**Comment by franticpsyx** [comment]
Hey @mazgor 

(1) This is a fundamental design decision: It's only on the single-product page that customer may decide whether they want to add the product to an existing subscription, or add it to the cart.

If you feel that your use case would benefit from a cart-item-level UI that lets customers add individual cart items to existing subscriptions, then the only option at this moment is to work with a developer to build it.

In short, when a product has already been added to the cart, there is no flow to add it to an existing subscription, unless cart subscriptions are active -- in which case the user may add the entire cart to an existing subscription.

(2) Sounds like a plugin/theme issue causing a JS error on the single-product page. Can't replicate the behavior you describe: https://cl.ly/352H281d1r2K
